src/Redux/books/books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Booksec = () => (
+  <>
+    <div className="books-container d-flex">
+      <h2>Book List</h2>
```

Comment:
- [ ] Please as it is a requirement for this project, add a `BookList` component that renders a list of books using the `Book` component, renders a form by using the `CreateNewBook` component

Reply:
Hello Mihndim2020,
The project requirement is about the initialization of react app and redux, I think what you are saying is for the next project